test/integration.js

```diff
@@ -0,0 +1,29 @@
+import 'should';
+import wiki from '../src/wiki';
+
+//TODO I leave this test skipped until I do the nock magic trick
+describe.skip('Dublin page integration', () => {
```

Comment:
Are you having a difficult time figuring out the nock stuff? I would rather get this test running and passing in Travis before we merge this.